### PR TITLE
Create a draft "working agreement"/"rules of engagement" document for the data engineering team.

### PR DESCRIPTION
**Pull request by richhaase**
cc: @bigbadace, @kschrodes, @roycehaynes @spucci @samirrayani

### CODE REVIEW

**Comment by richhaase** [comment]
Once everyone agrees with this basic structure/sentiment let's merge these changes and we can update via PR reviews like this going forward.

@bigbadace @kschrodes @roycehaynes @spucci @samirrayani

**Comment by spucci** [comment]
actually, I think this could live happily under "processes"

**Comment by richhaase** [comment]
@spucci done

**Comment by spucci** [comment]
👍

**Comment by kschrodes** [comment]
looks great

**Comment by roycehaynes** [comment]
Looks good. We should print that off and put it on our desk as a reminder. :-) 👍

File: Processes/Data Engineering.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+## The Team
+@bigbadace
+@kschrodes
+@richhaase
+@roycehaynes
+@spucci <- changing roles
+ 
+## On Call Rotation
+
+* [process] If you can't get to a page reach out to someone who can get to it.
```
Comment (bigbadace) [inline Processes/Data Engineering.md]: @richhaase does this mean - if i can't react to a page, call someone who can react? do i still acknowledge the alert even if i can't react to it?

Comment (richhaase) [inline Processes/Data Engineering.md]: @bigbadace i believe we decided that acknowledging a page was accepting responsibility for finding a resolution.  so if you can't react, as the on call person, you should find someone who can, and they should acknowledge the page.  you can even use victorops to reroute the page to the person accepting responsibility.

**Comment by mpetrovich** [comment]
This is really good. Thanks for putting this together @richhaase

File: Processes/Data Engineering.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+* [task] Change the on call rotation so that each person is secondary on call the week before they become primary.
+* [task] Change the on call rotation so that Sam and Samir are the 3rd tier of escalation for Data Engineering, Data Science and Web.
```
Comment (mpetrovich) [inline Processes/Data Engineering.md]: Consider using the exact same domain names for consistency & clarity: `Data Engineering, Research, and Web & API`. `Research` will need to be renamed to `Science` globally, but for now these docs should stay consistent.

File: Processes/Data Engineering.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    * Requires approval from other teams.
+* [task] Always make a ticket for every victorops issue that comes in while you are on call.
+    * [task] Tutorial on how to create a ticket in Jira
```
Comment (mpetrovich) [inline Processes/Data Engineering.md]: I'll own this part

File: Processes/Data Engineering.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+@roycehaynes
+@spucci <- changing roles
+ 
+## On Call Rotation
+
+* [process] If you can't get to a page reach out to someone who can get to it.
+* [task] Change the on call rotation so that each person is secondary on call the week before they become primary.
+* [task] Change the on call rotation so that Sam and Samir are the 3rd tier of escalation for Data Engineering, Data Science and Web.
+    * Requires approval from other teams.
+* [task] Always make a ticket for every victorops issue that comes in while you are on call.
```
Comment (mpetrovich) [inline Processes/Data Engineering.md]: 💯

File: Processes/Data Engineering.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    * [task] Tutorial on how to create a ticket in Jira
+
+## Alert Feedback
+
+* [process] Pair program when fixing alerts, and update the docs for the next engineer!
```
Comment (mpetrovich) [inline Processes/Data Engineering.md]: +∞

**Pull request merged**